frb_codegen/src/generator/rust/ty_struct.rs

```diff
@@ -154,6 +154,18 @@ impl TypeRustGeneratorTrait for TypeStructRefGenerator<'_> {
             Some(wrapper) => wrapper,
             None => &src.name,
         };
+        if src.is_empty {
+            return format!(
+                "impl support::IntoDart for {name} {{
+                    fn into_dart(self) -> support::DartAbi {{
+                        Vec::<u8>::new().into_dart()
+                    }}
+                }}
+                impl support::IntoDartExceptPrimitive for {name} {{}}
+                "
+            );
+        }
```

Comment:
nit: Is it possible we somehow merge this with the case below, the only different part is `vec![body].intodart()` vs `Vec::<u8>::new().intodart()`